src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass_Switch.cs

```diff
@@ -63,7 +63,8 @@ private void VisitSwitchBlock(BoundSwitchStatement node)
                 Join(ref afterSwitchState, ref this.State);
             }
 
-            if (reachableLabels.Contains(node.BreakLabel) || node.DefaultLabel == null && IsTraditionalSwitch(node))
+            if (node.DecisionDag.ReachableLabels.Contains(node.BreakLabel) ||
```

Comment:
>node.DecisionDag.ReachableLabels [](start = 16, length = 32)

Why can't we use the local here?

Reply:
We can, that was a mistake made when back porting this code from master, where that local is not defined.